azure/project.yml

```diff
@@ -4,3 +4,8 @@ variables:
   service_base_path: oauth2
   product_display_name: Identity Service
   product_description: Placeholder description
+  jinja_templates:
```

Comment:
Build appears to be complaining about this

Reply:
Fixed